clients/go/pkg/worker/jobDispatcher_test.go

```diff
@@ -188,13 +188,19 @@ func (suite *JobDispatcherSuite) completeJob(syncHandler bool) {
 
 type jobClientStub struct {
 	invoked bool
+	errorThrown bool
 }
 
 func (stub *jobClientStub) NewCompleteJobCommand() commands.CompleteJobCommandStep1 {
 	stub.invoked = true
 	return nil
 }
 
+func (stub *jobClientStub) NewThrowErrorCommand() commands.ThrowErrorCommandStep1 {
+	stub.errorThrown = true
```

Comment:
:x: As the property isn't used for the tests, you can just omit this.

The way the stub is used is mostly to make sure that the worker is doing what it's expected (i.e. it's completing jobs). So this function here can do like the `NewFailJobCommand`, i.e. `panic("implement me")`. This will make sure when we do want to use it, someone will have to write a proper implementation for the use case.

Reply:
thanks @[npepinpe](https://github.com/npepinpe)
i see the logic behind the panic, i thought it was an uncompleted code